pkg/sink/types.go

```diff
@@ -0,0 +1,52 @@
+package sink
+
+// Message is used to wrap the message written to the user defined sink
+// TODO: The numaflow project use sinksdk.Message as the input argument for Apply function.
+//
+//	Therefore, since the Message type doesn't have event time and watermark, these two fields
+//	won't be populated into the Datum.event_time and Datum.watermark.
+//	In short:
+//	  Add EventTime and Watermark in the Message type
+//	  or
+//	  Delete event_time and watermark in the sink proto file.
```

Comment:
Please check this TODO

Reply:
I feel like we only need Datum...
Message type is only used in the numaflow project. We can replace [sinksdk.Message](https://github.com/numaproj/numaflow/pull/174/files#diff-9bee584308ea77899e2541fc380d19957f6bde1e5f582d960d66609516f28556R46) with sinkpb.Datum.

Reply:
removed message type..